File: 8-kyu/Short Long Short/short-long-short.py
# Short Long Short
# https://www.codewars.com/kata/50654ddff44f800200000007/train/python

# def solution(a, b):
#   short = ''
#   long = ''
  
#   if len(a) > len(b):
#     long = a
#     short = b
#   else:
#     long = b
#     short = a
  
#   return short + long + short

def solution(a, b):
  return b + a + b if len(a) > len(b) else a + b + a

File: 8-kyu/Short Long Short/short-long-short.js
// Short Long Short
// https://www.codewars.com/kata/50654ddff44f800200000007/train/javascript

function solution(a, b) {
  const long = a.length >= b.length ? a : b;
  const short = a.length <= b.length ? a : b;

  // return short + long + short;
  return `${short}${long}${short}`;
}
